### PR TITLE
add gripper support

### PR DESCRIPTION
**Pull request by k-okada**
modified/cleanuped version of #23, this requires https://github.com/husky/husky/pull/22

### CODE REVIEW

File: husky_gazebo/launch/spawn_husky.launch [hunk 1]
```diff
@@ -53,6 +55,12 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
     <!-- Stow the arm -->
     <node pkg="husky_control" type="stow_ur5" name="stow_ur5"/>
 
+    <!-- Start the pr2 gripper controller and action server -->
+    <group if="$(arg gripper_enabled)">
+        <rosparam command="load" file="$(find husky_control)/config/control_gripper.yaml" />
```
Comment (paulbovbel) [inline husky_gazebo/launch/spawn_husky.launch:60]: Not seeing this file anywhere

File: husky_gazebo/package.xml [hunk 2]
```diff
@@ -27,6 +28,7 @@
   <run_depend>husky_control</run_depend>
   <run_depend>husky_description</run_depend>
   <run_depend>pointcloud_to_laserscan</run_depend>
+  <run_depend>pr2_controller_manager</run_depend>
```
Comment (paulbovbel) [inline husky_gazebo/package.xml]: Unneeded, you are using the ros_control controller_manager here.

File: husky_gazebo/launch/spawn_husky.launch (hunk 1 above)
Comment (k-okada) [inline husky_gazebo/launch/spawn_husky.launch:60]: forget to add see https://github.com/husky/husky/pull/22

File: husky_gazebo/package.xml (hunk 2 above)
Comment (k-okada) [inline husky_gazebo/package.xml]: fixed

File: husky_gazebo/launch/spawn_husky.launch [hunk 3]
```diff
@@ -53,6 +55,17 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
...
+        <node name="r_gripper_controller_spawner" pkg="controller_manager" type="spawner" args="r_gripper_controller"/>
+        <!-- https://github.com/ros-controls/ros_controllers/pull/181 -->
+        <node name="r_gripper_state_publisher" pkg="rostopic" type="rostopic"
```
Comment (paulbovbel) [inline husky_gazebo/launch/spawn_husky.launch:63]: Why is this necessary? Are these joints not controlled?

Comment (k-okada) [inline husky_gazebo/launch/spawn_husky.launch:63]: we need to publish joint_states for non actuated joints for moveit

## 

◉ Kei Okada

Comment (paulbovbel) [inline husky_gazebo/launch/spawn_husky.launch:63]: Understood, but does r_gripper_controller not control anything other than r_gripper_l_finger_joint? Are these joints just floating, or is there a mechanism to keep them rigid?

Comment (k-okada) [inline husky_gazebo/launch/spawn_husky.launch:63]: sending command to r_gripper_controller  change all joint in the gripper.

Comment (paulbovbel) [inline husky_gazebo/launch/spawn_husky.launch:63]: Understood, so then why is this necessary? Shouldn't the joint state controller publish the states for any controller joints?

Comment (k-okada) [inline husky_gazebo/launch/spawn_husky.launch:63]: not sure, but maybe there is not transmission for gripper joints

## 

◉ Kei Okada

**Comment by tonybaltovski** [comment]
Closing to due inactivity.
